web/packages/core/texts/en-US/messages.ftl

```diff
@@ -42,9 +42,12 @@ error-wasm-mime-type =
 error-invalid-swf =
     Ruffle cannot parse the requested file.
     The most likely reason is that the requested file is not a valid SWF.
-error-swf-fetch =
+error-swf-fetch-v2 =
```

Comment:
Why the addition of the `-v2`? Crowdin will throw out all the old translations regardless, and if you're not also keeping `error-swf-fetch` as a separate translation string I don't see a reason to change this key.

Reply:
> Crowdin will throw out all the old translations regardless

Huh? I didn't know that. That's actually not really good because this is utilising [a core Fluent feature](https://github.com/projectfluent/fluent/wiki/Fluent-vs-gettext#translation-invalidation): The separation of minor changes that don't affect translations and major changes that do affect them:
> Fluent's use of unique identifiers allows for at least two of those three states to remain separated - minor changes may be applied and if the developer does not alter the unique identifier (a.k.a does not change the social contract), all translations remain valid. On the other hand if the developer changes the ID, all translations become invalid and must be updated.

I wish Fluent would have implemented the idea of message versioning [they're investigating](https://github.com/projectfluent/fluent/wiki/Fluent-vs-gettext#translation-invalidation) as a third option because this is the ideal case in which this would be used (as old translations are still better than none, but should be marked as outdated and updated eventually). However, the [related issue](https://github.com/projectfluent/fluent/issues/141) hasn't been updated in years :/ I might try to get it more attention again in the future.

Reply:
As Crowdin doesn't follow this Fluent feature (which itself is not good imo), I removed the `-v2`.